dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/version/VersionPresenter.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * ******************************************************************************
+ *    Copyright 2016-2017 Spectra Logic Corporation. All Rights Reserved.
+ *    Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *    this file except in compliance with the License. A copy of the License is located at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file.
+ *    This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *    specific language governing permissions and limitations under the License.
+ * ******************************************************************************
+ */
+
+package com.spectralogic.dsbrowser.gui.components.version;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.dsbrowser.api.injector.ModelContext;
+import com.spectralogic.dsbrowser.api.services.logging.LogType;
+import com.spectralogic.dsbrowser.api.services.logging.LoggingService;
+import com.spectralogic.dsbrowser.gui.components.ds3panel.Ds3Common;
+import com.spectralogic.dsbrowser.gui.components.localfiletreetable.FileTreeModel;
+import com.spectralogic.dsbrowser.gui.services.jobService.factories.GetJobFactory;
+import com.spectralogic.dsbrowser.gui.util.BaseTreeModel;
+import com.spectralogic.dsbrowser.gui.util.DateTimeUtils;
+import com.spectralogic.dsbrowser.gui.util.treeItem.SafeHandler;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.*;
+import javafx.scene.layout.BorderPane;
+import kotlin.Pair;
+import kotlin.Unit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class VersionPresenter implements Initializable {
+    private static final Logger LOG = LoggerFactory.getLogger(VersionPresenter.class);
+
+    private final Ds3Common ds3Common;
+    private final GetJobFactory getJobFactory;
+    private final LoggingService loggingService;
+
+    @Inject
+    public VersionPresenter(
+            final Ds3Common ds3Common,
+            final GetJobFactory getJobFactory,
+            final LoggingService loggingService
+    ) {
+        this.ds3Common = ds3Common;
+        this.getJobFactory = getJobFactory;
+        this.loggingService = loggingService;
+    }
+
+    @FXML
+    private Button download;
+
+    @FXML
+    private TableView<VersionItem> versions;
+
+    @FXML
+    private BorderPane window;
+
+    @FXML
+    TableColumn<VersionItem, String> created;
+
+    @ModelContext
+    private VersionModel versionModel;
+
+    @Override
+    public void initialize(final URL location, final ResourceBundle resources) {
+        try {
+            final DateTimeUtils dateTimeUtils = new DateTimeUtils();
```

Comment:
I think I've seen this class injected in some places, but here a new one is constructed.  Which is the standard why we use this?